### PR TITLE
Create romanian quotes

### PR DESCRIPTION
**Pull request by DKparadox**
### Description

New and the first added quotes in Romanian

### CODE REVIEW

**Review by Miodec — CHANGES_REQUESTED** [review]
The file is missing the `.json` extension. Please add it.

**Review by Miodec — CHANGES_REQUESTED** [review]
Please rename the file to just `romanian.json`

**Comment by Ferotiq** [comment]
You still have a file just called `romanian`. Please delete

**Comment by DKparadox** [comment]
> You still have a file just called `romanian`. Please delete

did it

File: frontend/static/quotes/romanian.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "language": "romanian",
+  "groups": [
+    [0, 100],
+    [101, 300],
+    [301, 600],
+    [601, 9999]
+  ],
+  "quotes": [
+    {
+      "text": "Trecutul trebuie s? r?m?n?, ?n vie?ile noastre, la fel de plin de posibilit??i ca viitorul.",
```
Comment (Miodec) [inline frontend/static/quotes/romanian.json:11]: Are the question marks supposed to be there?

Comment (DKparadox) [reply frontend/static/quotes/romanian.json:11]: No. There were symbols like "ă", "â", "ș" and "ț". I suppose they have changed to question mark when I converted the file to .json .

Comment (Miodec) [reply frontend/static/quotes/romanian.json]: Go ahead and add those back then.

**Review by Miodec — APPROVED** [review]

**Pull request merged**